qa/automation/CareerPortal.Tests.Scenarios/Scenarious/All_Buttons.feature

```diff
@@ -0,0 +1,44 @@
+Background:
+Given I open the https://career.quantori.com/ portal in Google Chrome
+
+Scenario 1.0: Cookie Test
```

Comment:
It could be, yes, but I created these tests assuming we did not touch career portal before and wont touch it again :)
If we assuming that there will be a lot of test runs, regression testing like in reality when you are absolutely right)